contracts/entrypoint.sh

```diff
@@ -37,61 +37,4 @@ if [ "${DEPLOY_TYPE}" = "core" ]; then
         --force \
         --json \
         --via-ir
-
```

Comment:
Deleting these cases since the nomad file can just call forge commands inline and reduce complexity